lib/components/segmented-control/segmented-control.scss

```diff
@@ -0,0 +1,81 @@
+@import "../../styles/index.scss";
+
+.segmented-control {
```

Comment:
I think you need to add sliding animation https://www.figma.com/proto/Rgz5XTeUI4HaVWn0P0ACvr/branch/WAP76V7zN1B4byI1NZBrB3/Quill-components?page-id=22504%3A23&type=design&node-id=25738-2505&viewport=694%2C-440%2C0.23&review-number=25&t=hS52nXxZcpqmXGoL-9&scaling=min-zoom&starting-point-node-id=25738%3A2383&show-proto-sidebar=1

Reply:
yeah, i'm working on it, first want to get a preliminary review :)